src/main/java/com/spotify/annoy/ANNIndex.java

```diff
@@ -145,7 +147,7 @@ private float getNodeBias(final long nodeOffset) { // euclidean-only
   }
 
   public final float[] getItemVector(final int itemIndex) {
```

Comment:
Not really. itemIndex is the vector key and it is limited to the int size in the C++ version.